resque-scheduler.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.executables  = `git ls-files`.split("\n").map{|f| f =~ /^bin\/(.*)/ ? $1 : nil}.compact
   s.require_path = 'lib'
 
-  s.add_runtime_dependency 'redis', '>= 2.2.2', '< 3.1'
+  s.add_runtime_dependency 'redis', '>= 2.2.2', '< 3.2'
```

Comment:
Is there a reason this gem can't just have no Redis restriction?

Reply:
No reason that I'm aware of.  The only place I know of where we use our own fork of resque-scheduler is in backend, and of course we lock the Redis version there as well. 
 I'd be just as happy removing the restriction if you think that would be preferable.

FWIW, the mainline branch of this repo no longer has a max version for Redis.

Reply:
Now that I think of it, you had mentioned it would be fine for us to go back to mainline.  That might actually be the best option at this point.  I already have a PR open for the second attempt at the Redis upgrade in backend, so I'll see if that still passes CI using mainline.